src/components/Camera/index.js

```diff
@@ -38,7 +80,7 @@ export default class Camera extends Component {
     if (autoCapture) this.capture.start()
     events.on('onBeforeClose', () => {
       this.capture.stop()
-      route('/', true)
+      //route('/', true)
```

Comment:
not accidental, I was testing what the effect of it was and couldn't really see the difference, but now that I think of it, it might be for the modal logic.